### ISSUE TITLE
Convert 11G OCM Wiki Pages to Jekyll Format

### ISSUE THREAD

**Issue by dfhawthorne** [dfhawthorne/dfhawthorne.github.io#36]
Convert all Wiki pages under `docs/home/11g-ocm` to jekyll format

**Comment by dfhawthorne** [comment]
85 pages need to be converted as found by:

```bash
find_unconverted_web_pages.sh | \
    grep 11g-ocm | \
    wc -l
```

**Comment by dfhawthorne** [comment]
Wrote a script, `scripts/validate_issue_36.sh`, to validate conversion of web pages. The failures are listed as:

```text
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/GC.png.html failed
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/11g-ocm-set-up-and-configure-resource-manager/ASH_report_1327656054538.html?attredirects=0&d=1.html failed
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/11g-ocm-replay-a-captured-workload/workload_report_1329885258683.html?attredirects=0&d=1.html failed
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/11g-ocm-replay-a-captured-workload/workload_report_1329872485962.html?attredirects=0&d=1.html failed
```

These pages should be deleted during conversion.

**Comment by dfhawthorne** [comment]
I wrote a script, `scripts/convert_issue_36.sh`, to do the mass conversion/ The failures were:

```text
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/GC.png.html
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/GC.png.html is a New Google Sites page
Failed
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/11g-ocm-set-up-and-configure-resource-manager/ASH_report_1327656054538.html?attredirects=0&d=1.html
Unable to find the table containing the main content
Failed
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/11g-ocm-replay-a-captured-workload/workload_report_1329885258683.html?attredirects=0&d=1.html
Unable to find the table containing the main content
Failed
/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-performance-management/11g-ocm-replay-a-captured-workload/workload_report_1329872485962.html?attredirects=0&d=1.html
Unable to find the table containing the main content
Failed
```

**Comment by dfhawthorne** [comment]
I could not find a reference to `GC.png.html` in any converted web pages as shown by:

```bash
find_converted_web_pages.sh | \
    xargs grep -li GC.png.html | \
    wc -l
```

Nor in any unconverted web pages as shown by:

```bash
find_converted_web_pages.sh | \
    xargs grep -li GC.png.html | \
    wc -l
```

**Comment by dfhawthorne** [comment]
Five (5) issues were raised during the first conversion attempt:

1. #38: Converted 11G OCM Wiki Page Only Showing a Single Level TOC
1. #39: Empty 11G OCM Wiki Pages Found
1. #40: Link is javascript:void(0)
1. #41: Web Page Width is Too Wide on Reduction
1. #42: Incorrect Journal Link Found

**Comment by dfhawthorne** [comment]
Conversion of `docs/home/11g-ocm/11g-ocm-data-and-data-warehouse-management/11g-ocm-configure-flashback-data-archive.html` failed with the error message:

```text
Uploaded files directory ('/home/douglas/git-repositories/dfhawthorne.github.io/docs/home/11g-ocm/11g-ocm-data-and-data-warehouse-management/11g-ocm-configure-flashback-data-archive') does not exist
```

This was probably caused by the fix to issue #45 .

**Comment by dfhawthorne** [comment]
All 11G OCM pages have been converted.
